pkg/products/monitoring/dashboards/criticalSLOAlerts.go

```diff
@@ -505,8 +509,8 @@ const MonitoringGrafanaDBCriticalSLOAlertsJSON = `{
 			"scopedVars": {
 				"product": {
 					"selected": false,
-					"text": "redhat-rhmi-3scale|ThreeScale",
-					"value": "redhat-rhmi-3scale|ThreeScale"
+					"text": "` + integreatlyv1alpha1.RHMISpec{}.NamespacePrefix + `3scale|ThreeScale",
+					"value": "` + integreatlyv1alpha1.RHMISpec{}.NamespacePrefix + `3scale|ThreeScale"
```

Comment:
`integreatlyv1alpha1.RHMISpec{}.NamespacePrefix` doesn't look to resolve in grafana json either :thinking: 
![Screenshot from 2020-09-30 13-00-15](https://user-images.githubusercontent.com/24636860/94682477-e9b73e00-031c-11eb-94b6-5eb3389c13fd.png)

Reply:
same as above